database/interface.go

```diff
@@ -389,6 +389,15 @@ type Tx interface {
 	// implementations.
 	FetchBlockRegions(regions []BlockRegion) ([][]byte, error)
 
+	// PruneBlocks deletes the block files until it reaches the target size
+	// (specificed in bytes).
+	//
+	// The interface contract guarantees at least the following errors will
+	// be returned (other implementation-specific errors are possible):
+	//   - ErrTxNotWritable if attempted against a read-only transaction
+	//   - ErrTxClosed if the transaction has already been closed
+	PruneBlocks(targetSize uint32) ([][]byte, error)
```

Comment:
document return value. Chould be `[]chainhash.Hash`?

Reply:
Yes it could be `[]chainhash.Hash`. I'll update the code and document the return value